explorer/src/lib/components/map/Map.svelte

```diff
@@ -0,0 +1,57 @@
+<script>
+  // @ts-nocheck
+
+  import { geoNaturalEarth1, geoPath } from "d3";
+  import dataset from "./world-map.json";
+
+  import "./Map.css";
+
+  /** @type {Array<{lat: number, lon:number}> | Error}*/
+  export let nodes;
+
+  /** @type {import("d3").GeoProjection}*/
+  const projection = geoNaturalEarth1();
+
+  /** @type {import("d3").GeoPath<any, import("d3").GeoPermissibleObjects>} */
+  const path = geoPath(projection);
+
+  /** @param {Array<{lat:number, lon:number}> | Error} data */
+  function removeDuplicates(data) {
+    if (Array.isArray(data)) {
+      return data.filter(
+        (value, index, self) =>
+          index ===
+          self.findIndex((t) => t.lat === value.lat && t.lon === value.lon)
+      );
+    } else {
+      return [];
+    }
+  }
+
+  const markers = removeDuplicates(nodes);
+</script>
+
+<svg id="nodes-world-map" viewBox="0 0 954 477">
+  <defs>
+    <pattern
+      id="vertical-lines"
+      patternUnits="userSpaceOnUse"
+      width="5"
+      height="10"
+    >
+      <line x1="2.5" y1="0" x2="2.5" y2="10" stroke="black" stroke-width="1" />
+    </pattern>
+  </defs>
+  {#each dataset.features as data, index (index)}
+    <path d={path(data)} style="fill:url(#vertical-lines)" />
+  {/each}
+  {#if markers && markers.length !== 0}
```

Comment:
You can just do `{#if markers && markers.length}`. If markers.length is 0, then this becomes a falsy value.